Plugins/Json/src/DetectorJsonConverter.cpp

```diff
@@ -0,0 +1,176 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2023 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "Acts/Plugins/Json/DetectorJsonConverter.hpp"
+
+#include "Acts/Detector/Detector.hpp"
+#include "Acts/Detector/DetectorVolume.hpp"
+#include "Acts/Detector/Portal.hpp"
+#include "Acts/Navigation/DetectorVolumeFinders.hpp"
+#include "Acts/Plugins/Json/DetectorVolumeJsonConverter.hpp"
+#include "Acts/Plugins/Json/DetrayJsonHelper.hpp"
+#include "Acts/Plugins/Json/PortalJsonConverter.hpp"
+#include "Acts/Utilities/Enumerate.hpp"
+#include "Acts/Utilities/Helpers.hpp"
+
+#include <ctime>
+#include <set>
+
+nlohmann::json Acts::DetectorJsonConverter::toJson(
+    const GeometryContext& gctx, const Experimental::Detector& detector,
+    const Options& options) {
+  // Get the time stamp
+  time_t tt;
+  time(&tt);
+  auto ti = localtime(&tt);
```

Comment:
Could this be done using C++ `std::chrono`?

Reply:
That's what I wanted to use, but the formatting of day/year/etc is introduced only with `C++` as far as I could see.

Reply:
https://en.cppreference.com/w/cpp/io/manip/put_time maybe, but that also uses `localtime`

Reply:
Exactly, I thought I would change over to `std::chrome` when we switch to C++20.